doc/client_lua_api.md

```diff
@@ -736,6 +737,9 @@ Call these functions only at load time!
 ### Player
 * `minetest.get_wielded_item()`
     * Returns the itemstack the local player is holding
+* `minetest.send_chat_message(message)`
+    * Act as if `message` was typed by the player into the terminal.
+    * Commands starting with '/' are run.
```

Comment:
`* Commands starting with '/' are run.` isn't really necessary, plus it might mislead some people into thinking that command refers to  client-sided command not server-sided.